tools/c7n_azure/c7n_azure/session.py

```diff
@@ -149,7 +149,17 @@ def client(self, client):
         service_name, client_name = client.rsplit('.', 1)
         svc_module = importlib.import_module(service_name)
         klass = getattr(svc_module, client_name)
-        client = klass(self.credentials, self.subscription_id)
+
+        if sys.version_info[0] < 3:
```

Comment:
Can you assign these variables with a ternary operator:`klass_parameters` and `client`